packages/renderer/src/components/molecules/MarketOffersTable/MarketOffersTable.tsx

```diff
@@ -0,0 +1,144 @@
+// =============================================================================
+//  Copyright 2022 Haveno
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// =============================================================================
+
+import { createStyles } from "@mantine/core";
+import { createTable } from "@tanstack/react-table";
+import { useIntl } from "react-intl";
+import type { MarketOffer } from "./_types";
+import {
+  MarketOffersPriceCell,
+  MarketOffersAmountCell,
+  MarketOffersAccountTradesCell,
+  MarketOffersCostsCell,
+  MarketOffersAccountAgeCell,
+  MarketOffersPaymentCell,
+} from "./MarketOffersTableCell";
+import { Table } from "@molecules/Table";
+import { LangKeys } from "@constants/lang";
+import { TableVariant } from "@molecules/Table/_types";
+
+const table = createTable().setRowType<MarketOffer>();
+
+interface MarketOffersTableProps {
+  data: MarketOffer[];
```

Comment:
Global comment: `Array<Foo>` notation. This has been a recurring issue. Please make a note for future work.